src/Symfony/Component/Workflow/Definition.php

```diff
@@ -68,7 +66,7 @@ public function addPlace($place)
             throw new InvalidArgumentException(sprintf('The place "%s" contains invalid characters.', $place));
         }
 
-        if (!count($this->places)) {
```

Comment:
This is up for [debate](https://github.com/symfony/symfony/issues/19182) and maybe out of scope here..